config/repo-values.yaml

```diff
@@ -72,8 +72,10 @@ sync:
       url: https://hazelcast.github.io/charts/
     - name: nginx
       url: https://helm.nginx.com/stable
-    - name: cic
+    - name: citrix
       url: https://citrix.github.io/citrix-k8s-ingress-controller/
+    - name: citrix
```

Comment:
Names need to be unique for each repository. Can you please use two different unique names.